packages/terra-hyperlink/src/Hyperlink.module.scss

```diff
@@ -8,9 +8,12 @@
     outline: none;
     text-decoration: var(--terra-hyperlink-text-decoration, none);
 
-    &:visited,
-    &.is-visited {
+    &:visited {
```

Comment:
For better WCAG rules, change anywhere that uses this color `#0059a8` to `#006fc3`
```
6           color: var(--terra-hyperlink-color, #006fc3);
```

```
13          color: var(--terra-hyperlink-visited-color, #006fc3);
```

\
\
\
\
Plus add new theming variables for color when underline-hidden is applied:
```
  .is-underline-hidden {
    color: var(--terra-hyperlink-underline-hidden-color, #006fc3);
    text-decoration: none;
    
    &:focus {
      color: var(--terra-hyperlink-underline-hidden-focus-color, #001f67);
      text-decoration: underline;
    }

    &:hover {
      color: var(--terra-hyperlink-underline-hidden-hover-color, #001f67);
      text-decoration: underline;
    }

     &:visited,
     &.is-visited {
      color: var(--terra-hyperlink-underline-hidden-visisted-color, #006fc3);
      text-decoration: underline;

      &:focus {
        color: var(--terra-hyperlink-underline-hidden-visited-focus-color, #001f67);
      }
    }
  }
```

Reply:
Updated: https://github.com/cerner/terra-core/pull/2666/commits/bc8da28e26d17de029d9f6fcbd29808f669798a8.